app/middleware/middleware_healthcheck.rb

```diff
@@ -5,24 +5,28 @@ def initialize(app)
 
   def call(env)
     if env['PATH_INFO'.freeze] == '/status'.freeze
-      response = [200, { 'Content-Type' => 'application/json' }]
+      response = [503, { 'Content-Type' => 'application/json' }]
 
       # checks the database health
       begin
         db_healthy = ActiveRecord::Migrator.current_version != 0
-      rescue StandardError => e
-        Rails.logger.info "Database error: #{e}"
```

Comment:
was there a reason to remove this? seems like it would be helpful to log this error.

Reply:
I originally had it in there for debugging purposes, should I just keep it in there?

Reply:
I think it would be helpful if it ever actually happened